valhalla/baldr/graphtile.h

```diff
@@ -652,12 +653,13 @@ class GraphTile {
     // TODO(danpat): for short-ish durations along the route, we should fade live
     //               speeds into any historic/predictive/average value we'd normally use
 
-    constexpr double LIVE_SPEED_FADE = 1. / 3600.;
+    static double live_speed_fade = 1. / config().get<float>("baldr.live_speed_fading_sec", 3600);
```

Comment:
I'd keep the all caps for the variable, it's still pretty a constant, even if not a compile-time constant